packages/cli/test/helpers/confidence-interval.ts

```diff
@@ -0,0 +1,30 @@
+import { cartesianProduct, confidenceInterval } from '../../src/helpers/statistics/confidence-interval';
+import { expect } from 'chai';
+
+const control = [330.918433554412, 229.000245801897, 329.965804711059, 300.691712977023, 234.825814560758, 527.963570240824,
+  308.480228109542, 522.589636888846, 325.180019314907, 259.507753259425, 279.822702423339, 266.629000675101,
+  348.343075741739, 349.334465676962, 287.134434273056, 255.173593243992, 361.932565928192, 292.828093580835,
+  315.670479115113, 233.148959408031];
+
+const experiment = [311.307077576214, 287.01781844844, 326.260035401324, 354.377742876501, 288.289637886991, 273.101023005202,
+  274.56113018375, 315.970477279932, 219.797595818019, 347.564349614263, 273.942037169113, 346.293931534072,
+  245.535433540729, 289.58600732826, 283.287591042244, 329.580201801935, 182.369592581587, 256.401674519353,
+  260.634916843668, 274.456816887721, 294.182095864991, 273.935688437867, 248.996033572172, 322.630738439236,
+  250.809890805682, 229.394540671313, 326.040872321786, 235.165610985367, 239.857271090047, 347.272576200616];
+
+
+describe('confidence-interval test', () => {
+  it(`confidenceInterval()`, () => {
+    expect(confidenceInterval(control, experiment, 0.95)).to.equal([-8.466935463651964, 51.96882328768899]);
```

Comment:
If i globally install this module and run the `confidenceInterval` with the control and experiment data above the results i get are: 
`[ -78.158118168183, 3.9249323892729535 ]`

Confirming these assertions are correct?

Reply:
@krisselden what's the result suppose to be?

Reply:
should be `-8.466935463651964, 51.96882328768899`

https://gist.github.com/krisselden/d54a641a96410f48195209a85bc44c7c

Reply:
also, there is no reason this should be a separate function from the location shift estimate since it is the confidence interval of the location shift.